src/Voron/Voron.csproj

```diff
@@ -19,9 +19,15 @@
     <Compile Remove="Properties\AssemblyInfo.Windows.cs" />
     <Compile Include="..\CommonAssemblyInfo.Linux.cs" Link="Properties\CommonAssemblyInfo.Linux.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Data\CompactTrees\dictionary.bin" />
+  </ItemGroup>
   <ItemGroup>
     <Compile Include="..\CommonAssemblyInfo.cs" Link="Properties\CommonAssemblyInfo.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Data\CompactTrees\dictionary.bin" />
```

Comment:
What is the format for this? How was this generated?

Reply:
See comment below.

Reply:
What comment? This is generated with the usage of `Voron.Dictionary.Generator`, right?